File: 201/loop_python_ex/box_print.py
w = int(raw_input("Width? "))
h = int(raw_input("Height? "))
for a in range(0,h):
    if a == 0 or a == (h-1):
        print "*" * w
    else:
        print "*" + (" " * (w - 2)) + "*"

File: javascript/101/box_print.js
function printBox(width,height) {
  var s1 = "";
  var s2 = "";
  for (var i=0; i<width; i++) {
    s1 += "*";
    if (i===0|i===width-1) {
      s2 += "*";
    } else {
      s2 += " ";
    }
  }
  for (var j=0; j<height; j++) {
    if (j===0|j===height-1) {
      console.log(s1);
    } else {
      console.log(s2);
    }
  }
}

printBox(6,4);
